packages/faustwp-cli/src/index.ts

```diff
@@ -75,10 +75,9 @@ import { marshallTelemetryData, sendTelemetryData } from './telemetry/index.js';
   if (getWpSecret()) {
     try {
       const telemetryData = marshallTelemetryData(arg1);
-      void sendTelemetryData(telemetryData);
+      await sendTelemetryData(telemetryData);
     } catch (err) {
-      // console.log(err);
-      // Fail silently
+      console.log(err);
```

Comment:
What do you think about using our `debugLog` here so the error is only present when Faust debug mode is enabled?

```
import { debugLog } from './stdout/index.js';
```

Something like:

```suggestion
      debugLog(
      `Telemetry event failed: `,
      err,
    );
```